### PR TITLE
Added recipe for agate-lookup

### PR DESCRIPTION
**Pull request by pmlandwehr**
`agate-lookup` adds support for remote lookup tables to `agate`.

### CODE REVIEW

File: recipes/agate-lookup/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - agate >=1.5.0
+    - requests >=2.9.1
+    - pyyaml >=3.11
```
Comment (ocefpaf) [inline recipes/agate-lookup/meta.yaml]: Can you try removing the `run` time dependencies from the `build` section and see how it goes?

Comment (pmlandwehr) [inline recipes/agate-lookup/meta.yaml]: Do you mean the `build` deps? Like, [`source.py`](https://github.com/wireservice/agate-lookup/blob/master/agatelookup/source.py) imports all of the mentioned modules so I'm pretty sure they'll be needed...

Comment (ocefpaf) [inline recipes/agate-lookup/meta.yaml]: Nope, those are all used ar `run` time. I just built this with:

```yaml
requirements:
  build:
    - python
    - setuptools
  run:
    - python
    - agate >=1.5.0
    - requests >=2.9.1
    - pyyaml >=3.11
```

without any problems.

Comment (pmlandwehr) [inline recipes/agate-lookup/meta.yaml]: Ah, I misinterpreted. Sure, I can pull them. This has been an issue before.

**Comment by pmlandwehr** [comment]
The issue here is with `agate`: with no preinstall, we're getting a conflict in `py36`. I'll be rebuilding for `py36` once I've finished making the dependencies. At that point this build should work again.

**Comment by pmlandwehr** [comment]
ping @ocefpaf for review

**Pull request merged**